src/gsi/control2state.f90

```diff
@@ -1,3 +1,58 @@
+!-------------------------------------------------------------------------
+!    NOAA/NCEP, National Centers for Environmental Prediction GSI        !
+!-------------------------------------------------------------------------
+!BOP
+!
+! !MODULE:  control2state_mod --- control2state_mod variables and routines
+!
+! !INTERFACE:
+!
+module control2state_mod
+
+! !USES:
+
+
+! !DESCRIPTION: module control2state routines and variables
+
+use kinds, only: r_kind,i_kind
+use control_vectors, only: control_vector,c2sset_flg
+use gsi_4dvar, only: nsubwin, l4dvar, lsqrtb,ladtest_obs
```

Comment:
@jderber-NOAA 
Many of these variables are re-declared at the start of the subroutines.

Reply:
Good catch.  I think I started to check and then got distracted.   Updated.